app/models.py

```diff
@@ -632,9 +618,39 @@ def get_dict(self):
             "description": self.description,
             "hours": self.hours,
             "country_name": self.country_name,
-            "full_address": full_address
         }
 
+        shopDict["full_address"] = DiveShop.get_full_address(self.address1,
+        self.address2,
```

Comment:
can you tab all these in once so that it's clear that they're inside the function call